content/privacy.md

```diff
@@ -8,18 +8,13 @@ title = "Privacy"
 
 ## NATS Privacy Policy and Cookie Usage
 
-Effective Date: **September 15, 2018**
+Effective Date: **November 01, 2022**
 
 We are committed to safeguarding the privacy of our website visitors. In this policy we will explain how we treat your personal information.
-Analytics is used to track consumers of our site and improve its content. Various cookies along with Google Analytics are used to collect the following kinds of personal information.
 
-* Information about your computer and about your visits to and use of this website including your IP address, geographical location, browser type and version, operating system, referral source, length of visit, page views and website navigation paths.
-* Our website includes hyperlinks to, and details of, third party websites. We have no control over, and are not responsible for, the privacy practices of third parties.
-* We do not supply your personal information to third parties.
+We want to process as little personal information as possible when you use our website. That's why we've chosen Fathom Analytics for our website analytics, which doesn't use cookies and complies with the GDPR, ePrivacy (including PECR), COPPA and CCPA. Using this privacy-friendly website analytics software, your IP address is only briefly processed, and we (running this website) have no way of identifying you. As per the CCPA, your personal information is de-identified. You can read more about this on <a href="https://usefathom.com/" target="_blank">Fathom Analytics' website</a>.
```

Comment:
Where does this appear? Is it a pop up like GA?

Reply:
No, this is text on the /privacy page. The text was recommended by Fathom. All popups and banners were removed.
https://deploy-preview-784--nats-io.netlify.app/privacy/